<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab1">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Finder</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="3" column="1">
           <widget class="QPushButton" name="btnimgclr">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Clear Image</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btnnxt">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelcurrent">
            <property name="text">
             <string>Current</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btnprev">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="photoViewer">
            <property name="minimumSize">
             <size>
              <width>1</width>
              <height>1</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
                border: 4px dashed #aba
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="btnopen">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Open Directory</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelresult">
            <property name="text">
             <string>Total Results: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="photoMain">
            <property name="minimumSize">
             <size>
              <width>1</width>
              <height>1</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
                border: 4px dashed #aba
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>Settings - Index</string>
       </attribute>
       <widget class="QPushButton" name="SaveSettingsbtn">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>50</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Save Settings</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree Directory</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="VPTreeDir">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>381</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Hash Directory</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="HashDir">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>381</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QColumnView" name="columnView">
        <property name="geometry">
         <rect>
          <x>1000</x>
          <y>430</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Index Directory</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SelectFolderbtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Select Folder</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="IndexDir">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>210</y>
          <width>381</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="IndexGobtn">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>310</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
       <widget class="QLabel" name="IndexStatus">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>240</y>
          <width>631</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>170</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Update Index Database</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="PlaySoundCheck">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>240</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Sound</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Search Range</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="SearchRange">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>110</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Default: 10 (Works best in most cases)</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
